frontend/webadmin/modules/uicommonweb/src/main/java/org/ovirt/engine/ui/uicommonweb/models/storage/UploadImageModel.java

```diff
@@ -161,30 +160,32 @@ public void setTestResponse(EntityModel<Response> testResponse) {
     public UploadImageModel(final Guid limitToStorageDomainId, final DiskImage resumeUploadDisk) {
         if (resumeUploadDisk == null) {
             setDiskModel(new NewDiskModel() {
+                private final boolean isChangeable = limitToStorageDomainId == null;
```

Comment:
I really wonder whether it would be more readable to just turn this `isChangeable` data member into a local variable of the `UploadImageModel` C'tor defined inside `if (resumeUploadDisk == null)` before `setDiskModel()`. Not sure what's more readable and elegant :)

Reply:
Honestly, I'd avoid such changes, cause I'm afraid to break something in the model, and I'm fine with the way it is.
But I believe @rszwajko might know better than me.